alerting/src/main/kotlin/org/opensearch/alerting/TriggersActionThresholdUtils.kt

```diff
@@ -0,0 +1,38 @@
+package org.opensearch.alerting.util
```

Comment:
Need to add the license header here. [Example](https://github.com/opensearch-project/alerting/blob/main/alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt#L1-L4)